spec/models/like_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    it 'belongs to an author' do
+      association = described_class.reflect_on_association(:author)
+      expect(association.macro).to eq(:belongs_to)
+      expect(association.options[:class_name]).to eq('User')
+    end
+
+    it 'belongs to a post' do
+      association = described_class.reflect_on_association(:post)
+      expect(association.macro).to eq(:belongs_to)
+    end
+  end
+
+  describe 'callbacks' do
+    describe 'after_save' do
+      let(:user) { User.create(name: 'Sigh') }
+      let(:post) { Post.create(title: 'Test Post') }
+    end
+  end
+end
```

Comment:
Nice job so far. However, as part of the project requirements, any custom method in the models should have a corresponding unit test. Kindly consider to add a unit test for the `update_like_count` method in the like model here. :pray: